### PR TITLE
Tweak the model change to keep backward compatibility

### PR DESCRIPTION
**Pull request by ncloudioj**
r? @tspurway

### CODE REVIEW

File: migrations/versions/3cc07fb28766_.py [hunk 1]
```diff
@@ -8,7 +8,7 @@
 
 # revision identifiers, used by Alembic.
 revision = '3cc07fb28766'
-down_revision = '16fa35dd63a2'
+down_revision = None
```
Comment (tspurway) [inline migrations/versions/3cc07fb28766_.py]: isn't there a down migration?

Comment (ncloudioj) [inline migrations/versions/3cc07fb28766_.py]: Oops, that was a mis-kill. It's now fixed.

**Comment by tspurway** [comment]
r+

the migration script works on this, thanks, Nan!

**Comment by ncloudioj** [comment]
Awesome! Merging it now.

**Pull request closed without merging**